pipeline_logs/firestore_pipeline_logs_table.py

```diff
@@ -0,0 +1,37 @@
+import firebase_admin
+from core_data_modules.logging import Logger
+from firebase_admin import credentials, firestore
+
+
+log = Logger(__name__)
+
+
+class FirestorePipelinesLogsTable(object):
+
+    def __init__(self, cert):
+        """
+        :param cert: Path to a certificate file or a dict representing the contents of a certificate.
+        :type cert: str or dict
+        """
+        cred = credentials.Certificate(cert)
+        firebase_admin.initialize_app(cred, name="firestore_pipeline_logs_table")
+        self.client = firestore.client()
+
+
+    def _get_pipeline_log_doc_ref(self, pipeline_name, timestamp_string):
+        return self.client.document(f"metrics/pipeline_logs/{pipeline_name}/{timestamp_string}")
+
+
+    def update_pipeline_logs(self, pipeline_name, timestamp_string, pipeline_logs):
```

Comment:
Could you add a convenience function with a signature something like:
```python
def log_event(self, event: str)
```

or 
```python
def log_event(self, pipeline_name, run_id, event)
```
depending on your answer to another comment above

Reply:
added thanks.